staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -53,6 +53,33 @@ type AltNames struct {
 	IPs      []net.IP
 }
 
+// CleanAltNames cleans duplicate items in DNSNames and IPs.
+func CleanAltNames(altNames *AltNames) {
```

Comment:
in general i like the idea of cleaning up duplicate entries in the altNames.
but please name this function `RemoveDuplicateAltNames`.

also kubeadm supports wildcards such as `127.0.*.*` do we care to remove `127.0.0.1` if `127.0.*.*` is in the list already?
possibly not.

Reply:
RemoveDuplicateAltNames is indeed more clear.
For the time being, only the repetition of the character form is removed, and logical repetition may be necessary under observation.